fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/MfdFmsPerf.tsx

```diff
@@ -2505,12 +2506,23 @@ export class MfdFmsPerf extends FmsPage<MfdFmsPerfProps> {
                       <div style="display: flex; flex-direction: row;">
                         <span class="mfd-label mfd-spacing-right perf-appr-weather">BARO</span>
                         <InputField<number>
-                          dataEntryFormat={new AltitudeFormat(Subject.create(0), Subject.create(maxCertifiedAlt))}
+                          dataEntryFormat={new AltitudeFormat(Subject.create(1), Subject.create(maxCertifiedAlt))}
                           dataHandlerDuringValidation={async (v) => {
-                            SimVar.SetSimVarValue('L:AIRLINER_MINIMUM_DESCENT_ALTITUDE', 'feet', v);
+                            if (!this.props.fmcService.master?.fmgc.data.approachRadioMinimum.get()) {
+                              if (v === null) {
+                                SimVar.SetSimVarValue('L:AIRLINER_MINIMUM_DESCENT_ALTITUDE', 'feet', 0);
```

Comment:
I know this was here before, but can we move the simvar setting out of here? This should be abstracted away on some method maybe on `fmcService` or something. CC @flogross89

Reply:
I'm a new contributor here and I'm not sure if I should be doing these changes - It seems a bit difficult for me to do :D If you guide me, sure, I'll add it, otherwise I think you could open a new issue/PR which would target this.

Reply:
This would be well suited for ``FmcAircraftInterface``, but to be fair it's not consistently done that way. So I understand if you would not want to change it in this PR.